package.json

```diff
@@ -3,7 +3,7 @@
   "description": "Library for creating tagged constructors.",
   "repository": {
     "type": "git",
-    "url": "git://github.com/puffnfresh/daggy.git"
+    "url": "https://github.com/fantasyland/daggy.git"
```

Comment:
I changed it to match the protocol at https://github.com/fantasyland/fantasy-land/blob/3491c0f3c83d1fc82a6afac39ce56a089fd0ee82/package.json#L33